specs/trust-store-trust-policy.md

```diff
@@ -415,7 +415,7 @@ When delta CRLs are implemented, the following results can occur during revocati
 ##### OCSP Download
 
 OCSP URLs can be obtained from the certificate's authority information access (AIA) extension as defined in [RFC-2560](https://datatracker.ietf.org/doc/html/rfc2560).
-Notary V2 will wait for a default threshold of 5 seconds to receive an OCSP response.
+Notary Project specification implementation will wait for a default threshold of 5 seconds to receive an OCSP response.
```

Comment:
The Notary Project specification implementation

Reply:
update it to `Implementations of the Notary Project specification`